lua/presence/init.lua

```diff
@@ -483,13 +494,10 @@ function Presence:update_for_buffer(buffer, should_debounce)
         small_text = use_file_as_main_image and neovim_image_text or file_text,
     }
 
-    local editing_text = self.options.editing_text
-    editing_text = type(editing_text) == "function"
-         and editing_text(filename, buffer)
-         or string.format(editing_text, filename)
+    local status_text = self:get_status_text(filename, buffer)
```

Comment:
Since the `get_status_text` method only expects a single argument, we can remove the `buffer` variable passed here.
```suggestion
    local status_text = self:get_status_text(filename)
```